src/test/java/com/klarna/hiverunner/examples/HelloAnnotatedHiveRunnerTest.java

```diff
@@ -103,13 +103,6 @@
     }, encoding = "UTF-8")
     private HiveShell hiveShell;
 
-    
-    @BeforeEach
```

Comment:
Ah, so the `files` on `@HiveSql` above wasn't working so this method was doing it instead. Good find as this isn't necessary and isn't the recommended way of doing it.